packages/url/src/test/index.test.js

```diff
@@ -5,23 +5,23 @@ import { addQueryArgs } from '../';
 
 describe( 'addQueryArgs', () => {
 	test( 'should append args to an URL without query string', () => {
-		const url = 'https://andalouses.com/beach';
```

Comment:
😭 Just wanted to leave some of "my culture" in there (Not the domain, just the "andalouses" name) which is the name of beach not far from my place.

But don't worry I don't care that much